Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/ComputePass.h

```diff
@@ -78,9 +78,18 @@ namespace AZ
             // The draw item submitted by this pass
             RHI::DispatchItem m_dispatchItem;
 
-            // Whether or not to make the pass a full-screen compute pass. If set to true, the dispatch group counts will
-            // be automatically calculated from the size of the first output attachment and the group size dimensions.
-            bool m_isFullscreenPass = false;
+            // Whether or not to make the pass a full-screen compute pass, and which slot to use as the size source
+            AZ::Name m_fullscreenSizeSourceSlot;
```

Comment:
Can you change the variable name from m_fullscreenSizeSourceSlot to m_fullscreenSizeSourceSlotName to make it explicit that it's the name and not the slot?
This would help with code readability.

Reply:
done.